app/frontend/components/teamTile/teamTile.tsx

```diff
@@ -0,0 +1,27 @@
+import Link from 'next/link'
+import { TeamWithProjectsFragment } from '../../generated/graphql'
+
+export interface TeamTileProps {
+  team: TeamWithProjectsFragment
+}
+export const TeamTile = ({ team }: TeamTileProps): JSX.Element => {
```

Comment:
If I am not mistaken, you had already implemented this new `teamTile `in the older `teamChoice `component.

Reply:
I see no other TeamTile, gimme a hint?